tests/e2e/utils/prometheus/AMP-service-account.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: prometheus-server
+  name: amp-iamproxy-ingest-service-account
+  namespace: monitoring
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::<my-account-id>:role/amp-iamproxy-ingest-role
```

Comment:
Also we may be able to avoid creating this file by using

```
eksctl create iamserviceaccount  --name amp-iamproxy-ingest-service-account --namespace monitoring --cluster $CLUSTER_NAME --attach-policy-arn ${AMP_IAMPROXY_INGEST_POLICY} --override-existing-serviceaccounts   --approve --region $CLUSTER_REGION
```

This way we just have to create the policy and not the role, which could simplify the setup.